app/internal/page/data.go

```diff
@@ -80,14 +80,14 @@ const (
 type FeeType uint8
 
 const (
-	HalfFee FeeType = iota + 1
+	FullFee FeeType = iota + 1
```

Comment:
this needs to be `iota` so it starts off at 0 and works as the default, you also will want to change the generator comment not to have `-empty` (and `-trimprefix` isn't needed as the names aren't prefixed)

Reply:
the underlying field is `uint8` which has the default value of `0`, so basically you just need to give the 0 value a name